low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -252,7 +252,7 @@ def _check_mismatch_count(self, device):
                         faulty_device = data.split(":")[0].rstrip()
                         if device == faulty_device:
                             self.alert_type = self.FAULT_RESOLVED
-                            self._alert_msg = "Mismatch_cnt found '0' for " + device
+                            self._alert_msg = "Prior data inconsistency fault detected in RAID protected disks, in %s RAID array is resolved." %device
```

Comment:
Gregory has mentioned to avoid using inconsistent word in any of the message.

Reply:
@ujjwalpl 
can you please check and suggest what will be the proper msg :
For FAULT: 
1. **Data redundancy check failure for server disks in %d RAID array, disk needs re-synchronization.**
OR
2. **RAID protected disks present in %s RAID array, needs to be re-synchronized.**

For FAULT resolved:
1. **Resolved data redundancy error for server disks in %d RAID array**
OR
2. **RAID protected disks present in %s RAID array are synchronized.**



Reference:  

Derived from information given in https://www.kernel.org/doc/html/latest/admin-guide/md.html
>check
A full check of redundancy was requested and is happening. This reads all blocks and checks them. A repair may also happen for some raid levels.

Reply:
@ujjwalpl please confirm when time allows, we discussed and came up with above two options

Reply:
RAID disks present in %s RAID array, needs synchronization.
RAID disks present in %s RAID array are synchronized.

Reply:
@ujjwalpl  Thank You.
Updated msg. Please review.